src/device_info.sh

```diff
@@ -120,28 +121,39 @@ function get_cpu()
   flag=${flag:-'SILENT'}
   cmd_model="lscpu | grep 'Model name:' | sed --regexp-extended 's/Model name:\s+//g' | cut --delimiter=' ' -f1"
   cmd_frequency="lscpu | grep MHz | sed --regexp-extended 's/(CPU.*)/\t\t\1/'"
+  cmd_architecture="lscpu | grep 'Architecture:' | sed --regexp-extended 's/Architecture:\s+//g' | cut --delimiter=' ' -f1"
 
   case "$target" in
     1) #VM_TARGET
       cpu_model='Virtual'
+
+      show_verbose "$flag" "$cmd_architecture"
+      cpu_architecture=$(cmd_manager 'SILENT' "$cmd_architecture")
       ;;
     2) # LOCAL_TARGET
       show_verbose "$flag" "$cmd_model"
       cpu_model=$(cmd_manager 'SILENT' "$cmd_model")
 
       show_verbose "$flag" "$cmd_frequency"
       cpu_frequency=$(cmd_manager 'SILENT' "$cmd_frequency")
+
+      show_verbose "$flag" "$cmd_architecture"
+      cpu_architecture=$(cmd_manager 'SILENT' "$cmd_architecture")
       ;;
     3) # REMOTE_TARGET
       show_verbose "$flag" "$cmd_model"
       cpu_model=$(cmd_remotely "$cmd_model" 'SILENT')
 
       show_verbose "$flag" "$cmd_frequency"
       cpu_frequency=$(cmd_remotely "$cmd_frequency" 'SILENT')
+
+      show_verbose "$flag" "$cmd_architecture"
+      cpu_architecture=$(cmd_manager 'SILENT' "$cmd_architecture")
```

Comment:
you must use `cmd_remotely`.

Reply:
Don't forget about this! Do this change also in this commit (second)